CppCoreGuidelines.md

```diff
@@ -5090,7 +5090,7 @@ Leaving behind an invalid object is asking for trouble.
 ##### Example, bad
 
     class X3 {     // bad: the constructor leaves a non-valid object behind
-        FILE* f;   // call init() before any other function
```

Comment:
As far as I can tell, the intent of this comment was to say "call is_valid() before any other function", as demonstated lower with `if (file.is_valid()) { file.read(); `

Reply:
Added it